vmdb/app/models/ems_refresh/refreshers/kubernetes_refresher.rb

```diff
@@ -0,0 +1,14 @@
+module EmsRefresh::Refreshers
+  class KubernetesRefresher < BaseRefresher
+    include RefresherRelatsMixin
+    include EmsRefresherMixin
+
+    def initialize
+    end
```

Comment:
You don't need this method (unless you are intentionally killing the inherited initializer?)